src/components/bookList.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import BookItem from './bookItem';
+
+const BookList = () => (
+  <ul>
+    <BookItem />
+  </ul>
+);
+
+export default BookList;
```

Comment:
- [ ] It would be better if the `bookList` component should render a **LIST OF BOOKS** using the `BookItem` component. 😉